pwa-shell/src/ui/notifications/NotificationsHeader.js

```diff
@@ -1,31 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import { FormattedMessage, defineMessages } from 'react-intl.macro';
 
-const text = amount => {
-  switch(amount) {
-    case 0: return 'Non ci sono nuove notifiche';
-    case 1: return (
-      <React.Fragment>
-        Hai <span className="NotificationsHeader__amount">1</span> nuova notifica
-      </React.Fragment>
-    );
-    default: return (
+const messages = defineMessages({
+  headerNotifMessage: {
+    id: 'notification.headerNotifMessage',
+    defaultMessage: '{count,plural,=0{No new notifications}one{You have a new notification}other{You have # new notifications}}',
+  },
+});
+
+const text = (amount, caption) => {
```

Comment:
Please double check `react-intl` API, I think we can make this code way simpler with a single usage of `FormattedMessage`

Reply:
sorry I couldn't find a way on `react-intl` to use easier plural methods for our message dictionary `it.json` so I improvise it using string stuff functions i personally made. Please check the changes related to this and let me know what you think.